solver/src/driver.rs

```diff
@@ -261,9 +297,15 @@ fn liquidity_with_price(
     liquidity
 }
 
-struct SolverSettlement {
+// A single solver produces multiple settlements
+struct SolverSettlements {
     // Index in the Driver::solver vector
     index: usize,
+    settlements: Vec<SolverSettlement>,
+}
+
+// Each individual settlement has an objective value.
+struct SolverSettlement {
     settlement: Settlement,
     objective_value: BigRational,
 }
```

Comment:
I am unsure about these structs. They will most likely change when settlements get merged so I didn't think too hard about proper naming and encapsulation.